setup.py

```diff
@@ -33,8 +34,9 @@
         'License :: OSI Approved :: BSD License',
         'Programming Language :: Objective C',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
```

Comment:
Python 3.7 isn't released yet, the release is [scheduled for 2018-06-27](https://www.python.org/dev/peps/pep-0537/#schedule), so it may be too early to officially support it.